Resources/views/ContentType/view_content_type.html.twig

```diff
@@ -6,103 +6,154 @@
+                    <div class="ez-contenttype-metadata-name">
+                        {{ "content_type.default_availability"|trans }}
+                        <span class="ez-contenttype-metadata-help">{{ "content_type.default_availability.help"|trans|desc("Default availability in primary language, if translation is absent") }}</span>
+                    </div>
+                    <div class="ez-contenttype-metadata-value">{{ 'content_type.default_availability.value'|transchoice(content_type.defaultAlwaysAvailable)|desc("{0} Not available|{1} Available") }}</div>
+                </div>
+            </section>
+
+            <section>
+                <h2 class="ez-contenttype-title">{{ 'content_type.content.title'|trans|desc('Content field definitions') }}</h2>
+
+                {% for group, field_definitions in fielddefinitions_by_group %}
+                    <section class="ez-field-definition-group">
+                        <h3 class="ez-contenttype-subtitle">{{ group|trans([], 'ezplatform_fields_groups') }}</h3>
+                        <table class="ez-field-definitions">
+                            <thead>
+                                <tr>
+                                    <th class="ez-field-definition-header">{{ "field_definition.name"|trans|desc("Name") }}</th>
+                                    <th class="ez-field-definition-header">{{ "field_definition.identifier"|trans|desc("Identifier") }}</th>
+                                    <th class="ez-field-definition-header">{{ "field_definition.type"|trans|desc("Type") }}</th>
+                                    <th class="ez-field-definition-header"></th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                            {% for field_definition in field_definitions %}
+                                <tr>
+                                    <td class="ez-field-definition-main-properties">{{ field_definition.name(language_code) }}</td>
+                                    <td class="ez-field-definition-main-properties">{{ field_definition.identifier }}</td>
+                                    <td class="ez-field-definition-main-properties">{{ field_definition.fieldTypeIdentifier }}</td>
+                                    <td class="ez-field-definition-main-properties"></td>
+                                </tr>
+                                <tr>
+                                    <td colspan="4">
+                                        <ul class="ez-field-definition-properties">
+                                            <li class="ez-field-definition-property">
+                                                <div class="ez-field-definition-property-name">{{ 'field_definition.description'|trans|desc("Description") }}</div>
+                                                <div class="ez-field-definition-property-value">{#{ field_definition.description(language_code) }#}</div>
```

Comment:
@bdunogier Was there any other reason why it was commented out or should proposed solution above work to your knowledge as well?